apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -18,111 +18,172 @@ export function InstructorCourseAdminSettings({
+                <label for="repository">Repository</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="repository"
+                  name="repository"
+                  value="${resLocals.course.repository}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  The Github repository that can be used to sync course files.
+                </small>
+              </div>
+              <p class="mb-0">
+                ${coursePathExists && courseInfoExists
+                  ? resLocals.authz_data.has_course_permission_view
+                    ? resLocals.authz_data.has_course_permission_edit &&
+                      !resLocals.course.example_course
+                      ? html`
+                          <a
+                            data-testid="edit-course-configuration-link"
+                            href="${resLocals.urlPrefix}/${resLocals.navPage}/file_edit/infoCourse.json"
+                          >
+                            Edit course configuration
+                          </a>
+                          in <code>infoCourse.json</code>
+                        `
+                      : html`
+                          <a
+                            href="${resLocals.urlPrefix}/${resLocals.navPage}/file_view/infoCourse.json"
+                          >
+                            View course configuration
+                          </a>
+                          in <code>infoCourse.json</code>
+                        `
+                    : ''
+                  : ''}
+              </p>
+            </form>
           </div>
         </main>
       </body>
     </html>
   `.toString();
 }
+
+function CourseDirectoryMissingAlert({
+  resLocals,
+  coursePathExists,
+  courseInfoExists,
+}: {
+  resLocals: Record<string, any>;
+  coursePathExists: boolean;
+  courseInfoExists: boolean;
+}) {
+  if (resLocals.authz_data.has_course_permission_edit && !resLocals.course.example_course) {
+    if (!coursePathExists) {
+      return html`
+        <div class="alert alert-danger">
+          Course directory not found. You must
+          <a href="${resLocals.urlPrefix}/${resLocals.navPage}/syncs"> sync your course </a>
+          .
+        </div>
+      `;
+    } else if (!courseInfoExists) {
+      return html`
+        <form name="add-configuration-form" method="POST" class="alert alert-danger">
+          <code>infoCourse.json</code> is missing. You must
+          <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+          <button
+            name="__action"
+            value="add_configuration"
+            class="btn btn-link btn-link-inline mt-n1 p-0 border-0 "
+          >
+            create this file
+          </button>
+          to edit your course settings.
+        </form>
+      `;
+    } else {
+      return;
```

Comment:
This is unnecessary.

```suggestion
```

Reply:
Removed locally in [8e3b7a9](https://github.com/PrairieLearn/PrairieLearn/pull/9386/commits/8e3b7a92c2e5562595292f74ea7fb09cb6932da6).